src/Carter/OpenApi/CarterOpenApi.cs

```diff
@@ -368,9 +371,19 @@ private static void AddSchema(Dictionary<string, SchemaElement> navigation, Open
                 
                 foreach (var memberKeyValue in keyValuePair.Value.DataMembers)
                 {
-                    var propertySchema = SchemaFromElement(memberKeyValue.Value);
                     var propertyInfo = keyValuePair.Value.ElementType.GetProperties().Where(o => CamelCase(o.Name) == memberKeyValue.Key).SingleOrDefault();
                     object[] attribute = propertyInfo.GetCustomAttributes(typeof(ApiSchemaAttributes), true);
+
+                    if (options.SchemaIgnoreAttribute != null)
```

Comment:
Won't this always be not null?

Reply:
You can set the option to null to turn off the feature.  I did it like that for backwards compatibility.  But thinking on it now who would use the attribute and expect it not to get ignored?